src/CMakeLists.txt

```
@@ -105,33 +115,45 @@ target_include_directories(
 
     PUBLIC
     ${PROJECT_SOURCE_DIR}/include
-    ${PROJECT_SOURCE_DIR}/third_party
     # Include configured header file which contains engine metadata
     ${CMAKE_CURRENT_BINARY_DIR}/include/
 )
 
-if(CONAN_LIBS)
-    target_link_libraries(
-        inexor-vulkan-renderer
-
-        PUBLIC
-        ${CONAN_LIBS}
-    )
-else()
-    foreach(_LIB ${CONAN_LIBS_RELEASE})
-        target_link_libraries(
-            inexor-vulkan-renderer
-            optimized
-
-            ${_LIB}
-        )
-    endforeach()
-    foreach(_LIB ${CONAN_LIBS_DEBUG})
-        target_link_libraries(
-            inexor-vulkan-renderer
-            debug
-
-            ${_LIB}
-        )
-    endforeach()
+# Declare use of dependencies
+FetchContent_MakeAvailable(fmt)
+FetchContent_MakeAvailable(glfw)
+FetchContent_MakeAvailable(glm)
+FetchContent_MakeAvailable(imgui)
+add_library(imgui
+    ${imgui_SOURCE_DIR}/imgui.cpp
+    ${imgui_SOURCE_DIR}/imgui_draw.cpp
+    ${imgui_SOURCE_DIR}/imgui_tables.cpp
+    ${imgui_SOURCE_DIR}/imgui_widgets.cpp)
+target_include_directories(imgui PUBLIC ${imgui_SOURCE_DIR})
+FetchContent_MakeAvailable(spdlog)
+FetchContent_MakeAvailable(stb)
+FetchContent_MakeAvailable(tinygltf)
+FetchContent_MakeAvailable(toml)
+FetchContent_MakeAvailable(vma)
+FetchContent_MakeAvailable(volk)
+FetchContent_MakeAvailable(Vulkan)
+
+# Work around a VMA compile error from its use of snprintf
+target_compile_definitions(VulkanMemoryAllocator PRIVATE VMA_STATS_STRING_ENABLED=0)
+
+if (WIN32)
+   set(VOLK_STATIC_DEFINES VK_USE_PLATFORM_WIN32_KHR)
 endif()
+
+target_link_libraries(inexor-vulkan-renderer
+    PUBLIC
+    fmt::fmt
+    glfw
+    glm::glm
+    imgui
+    spdlog
+    tinygltf
+    toml11::toml11
+    volk
```

```suggestion
    spdlog::spdlog
    tinygltf
    toml11::toml11
    volk::volk
```
We should use the aliased target names where possible.

---

Done.